### ISSUE TITLE
Question about running intern tests

### ISSUE THREAD

**Issue by ianwhite** [turbolinks/turbolinks#455]
Hello, I'm attempting to write a new feature branch, and have managed to get my head around typescript and the new test framework.  I've got some failing tests, which is a good start.

However, I'd like to do some console.log debugging, and can't figure out how to access the console in the tests.  I've seen that you can start intern in browser mode with `yarn start`, but I cannot access any of the tests there.

Can you provide any hints please?

**Comment by domchristie** [comment]
Hi @ianwhite 

>  I've got some failing tests, which is a good start.

🎉 😄 

> However, I'd like to do some console.log debugging, and can't figure out how to access the console in the tests. 

IIRC I think I may have come across the same issue. Rather than debug the test suite, I think I added `console.log` to the source where necessary. I then simulated the test code manually by running `yarn start`, opening `http://localhost:9000/fixtures/navigation.html` (for example) in a browser window, and clicking around. I hope that makes sense?
